0.0.4

Arzte opened the pull request:
Its about time we bump the version, TODO stop slacking on version updates.

Arzte committed:
Stop the muppets spamming !stats

Its really annoying me, can't stand it anymore.

Arzte committed:
Remove comment for uneeded code

If for some weird reason it turns out we need this, this commit can always be reverted.

Arzte committed:
Merge branch 'master' into develop

Arzte committed:
Open DM's correctly for !stats

I made a newb mistake, man its been a bit since I've made anything for MB,

Arzte committed:
Delete !stats command message [MUST MINIMIZE SPAM]

In yet another line of mistakes I didn't consider deleting the command after it was sent like a fucking idiot.

Arzte committed:
Merge branch 'hotfix/remove_more_stats_spam' into develop

SteamingMutt commented:
Bump the version in package.json too.

SirDoctors commented:
👍

Dougley commented on Commands/Commands/utils.js:
```diff
@@ -87,6 +87,8 @@ commands.stats = {
           fields: field
         }).catch(bugsnag.notify) // Send Message to DM error
+      }).then(successmsg => {
+        setTimeout(() => bot.Messages.deleteMessages([msg]), config.timeouts.messageDelete)
```
Can't bulk-delete a single message.
If this is the `!stats` message, just delete it instantly.

Dougley replied:
If this is the result that got send to your DMs,  deleting isn't necessary.

Arzte committed:
Update package version and info's inside it.

Arzte committed:
Delete the !stats command message the proper way

The former way is for bulk delete, which needs more than one message to delete.

Dougley commented on package.json:
```diff
@@ -15,7 +15,14 @@
   "maintainers": [
     {
-      "name": "Benjamin Hoffmeyer (TheDoctorsLife)"
+      "name": "Remco Jongschaap (SteamingMutt)",
```
I can't be both the author and a maintainer 😉

Arzte replied:
👀
